dns/checkip/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		checkip
+PLUGIN_VERSION=		1.0
+PLUGIN_REVISION=	0
+PLUGIN_COMMENT=		Use alternate services to get current IP address (custom DynDNS and RFC2136).
+PLUGIN_MAINTAINER=	NOYB at github
```

Comment:
this should be an email address

Reply:
Email addresses change.  Github account is better contact channel than a hard coded email address.